lib/transcoder.ex

```diff
@@ -69,16 +69,31 @@ defmodule Membrane.Transcoder do
                 and is supposed to return the desired output stream format or its module.
                 """
               ],
-              force_transcoding?: [
-                spec: boolean() | (stream_format() -> boolean()),
-                default: false,
+              transcoding_policy: [
+                spec:
+                  :always
+                  | :if_needed
+                  | :never
+                  | (stream_format() -> :always | :if_needed | :never),
+                default: :if_needed,
                 description: """
-                If set to `true`, the input media stream will be decoded and encoded, even
-                if the input stream format and the output stream format are the same type.
+                Specifies, when transcoding should be appliead.
 
                 Can be either:
-                * a boolean,
-                * a function that receives the input stream format and returns a boolean.
+                * an atom: `:always`, `:if_needed` (default) or `:never`,
+                * a function that receives the input stream format and returns an atom.
+
+                If set to `:always`, the input media stream will be decoded and encoded, even
+                if the input stream format and the output stream format are the same type.
+
+                If set to `:if_needed`, the input media stream will be transcoded only if the input
+                stream format and the output stream format are different types.
+                This is the default behavior.
+
+                If set to `:never`, the input media stream won't be neither decoded nor transcoded.
```

Comment:
It won't be `encoded` either, for instance in scenario with `input: RawVideo` and `output: H264`